### ISSUE TITLE
specify call for wide review end date

### ISSUE THREAD

**Issue by nigelmegitt** [w3c/spec-releases#3]
It'd be helpful for WD intended for wide review to be able to put in the review end date as specified in the SOTD (_if_ specified!), as well as any real world expected comment processing period, and have the tool calculate the earliest publication date for a CR based on that.

**Comment by nigelmegitt** [comment]
Plus, if you also allow the last intended WD for WR publication date to be entered, it could calculate the earliest recommended review end date.

As an aside, I _think_ the Process section §6.4 Candidate Recommendation says that MUST be at least 4 weeks after publication, but it is a bit odd that the 4 weeks isn't in the §6.2.3.1 Wide Review section. Being placed in §6.4 makes it unclear if the CR version itself has to have a comment window open for at least 4 weeks (in addition to the concluded wide review period) or if, prior to publishing the CR, a comment window must have been open for at least 4 weeks as part of the wide review solicitation.

**Comment by plehegar** [comment]
The process doesn't set expectation on end dates for wide review. It just expects the wide review to be done before the move to CR. How about I add a "last day" to request wide review, which would be around 4 weeks before the transition request to CR.

**Comment by nigelmegitt** [comment]
Yes, a last day for WR to begin would be fine. It would additionally be useful to be able to enter the group's planned duration for disposing of any review comments. The way I see it:

* Group requests wide review, optionally (but it's good practice) setting an end date
* Wide review period ends (minimum 4 weeks after review was requested
* Group disposes of comments (typically takes some weeks unless there are no comments or there is a super fast working model)
* Group requests transition to CR

**Comment by plehegar** [comment]
I can add a box to change the amount of time of responding to comments after the wide review and the CR end date.

**Comment by plehegar** [comment]
@nigelmegitt , I added computation for:
* 10 days CfC (per group charter)
* 3 months wide review before CR (also per Group charter).
* and a setting at the end to allow weeks to address comments after the wide review

Let me know if this solves this.

**Comment by nigelmegitt** [comment]
Thanks @plehegar that sounds good. Not sure if you did 10 calendar days or 10 working days for the CfC period? Certainly for TTWG, it's working days, and we usually translate that into 14 calendar days.

**Comment by plehegar** [comment]
Good catch. Looking back at the charter template, it does say "working days". I'll update the tool before closing this issue.

**Comment by plehegar** [comment]
actually, the charter template says "one week to 10 working days". Timed Text picked 10 working days. So, it will have to be a parameter and the tool will need to load the existing charter to attempt to guess which default to apply.
